### ISSUE TITLE
Sort and filters

### ISSUE THREAD

**Issue by jacobtomlinson** [mattvh/jekyllthemes#104]
Thanks for this project, I find it really helpful.

Themes on your site are just ordered chronologically, have you thought about giving different sort options?

I imagine it would be possible to sort by number of GitHub stars using JavaScript.

**Comment by gynter** [comment]
I propose the sorting features by date and alphabetically from ascending to descending and vice versa.

The filtering imho is a bit of harder to do - Yes, it can be done by using tags, but what exactly are those tags going to be so the filters won't get to noisy? Something like *one-page*, *dark*, *light*, *sidebar*, *fixed-navigation* etc?

**Comment by jacobtomlinson** [comment]
I agree about the filters. Really it would be up to the moderator/owner to manage the tags to keep things simple, so there would have to be buy in from him.

**Comment by nmenag** [comment]
sort by categories

**Comment by gynter** [comment]
@nandomoreirame I'd be happy if You could contribute by making the sorting and filtering feature (You mentioned it in #204). My proposal is still the same as above:

> I propose the sorting features by date and alphabetically from ascending to descending and vice versa.

> The filtering imho is a bit of harder to do - Yes, it can be done by using tags, but what exactly are those tags going to be so the filters won't get to noisy? Something like one-page, dark, light, sidebar, fixed navigation, one column two columns, three columns etc? The tags should be kept as generic as possible.

**Comment by nandomoreirame** [comment]
Hey guys!
I can propose a new layout?

Tks

**Comment by nandomoreirame** [comment]
### Home
![jekyllthemes-home](https://cloud.githubusercontent.com/assets/1318271/12374727/34f91064-bc8c-11e5-93e7-eec17dcfd890.png)

### All themes
![jekyllthemes-all](https://cloud.githubusercontent.com/assets/1318271/12374777/f72f9ad0-bc8d-11e5-8eee-aaa13fa5e939.png)

**Comment by nandomoreirame** [comment]
### I can do:

* [x] Search Themes
* [x] Order by date and title
* [x] Filter by Category
* [x] Filter by Tag
* [x] Filter by Author

:metal:

**Comment by nandomoreirame** [comment]
@gynter and @mattvh :smiley:

**Comment by gynter** [comment]
Since I'm not the lead for the project I'm not authorized to make huge changes in the repository (changing the layout is considered a major change). We have to wait @mattvh approval for this.

**Comment by iBobik** [comment]
Proposed filters:
- usage: blog, CV, product presentation, commerce, portfolio, gallery, …
- features: responzive, animations, fluid, form elements, parallax, BEM, SMACSS, …
- libraries/tools used: Bootstrap, Zurb, PostCSS, LESS, SASS

It is good to consider how to motivate or enforce theme authors to fill it. Maybe tests, so it will be visible in pull request if something is not filled?

Also bigger screenshot helps (up to full screen, switch for size).

**Comment by ohadschn** [comment]
Filtering/sorting by GitHub forks (not just stars) would be great as well.

**Comment by yodrew** [comment]
I would like this, too. As it is, the site is useless to me. I want to be able to search for themes that would work for my purpose structurally, then choose according to appearance. For example, I need theme to handle pages and sub pages, slide out menu. Then I would look at search results for right sidebar, dark background. Filtering by structure because floating a sidebar from left to right is easy compared to making a layout and _config file to work with it. Theme creators should also be able to add categories to filter by.

**Comment by awareness481** [comment]
@mattvh Would you be interested in an initial implementation and see if we/you can build from there? I am interested in contributing
